src/galasm.c

```diff
@@ -575,18 +575,6 @@ int AssemblePldFile(char *file, struct Config *cfg)
                                 }
                             }
                         }
-
-                        /* output and feedback? then mode 2 */
-
-                        if (OLMC[n].PinType == COM_TRI_OUT && OLMC[n].FeedBack)
-                        {
-                            modus = MODE2;      /* mode 2 */
-
-                            Jedec.GALSYN = 1;   /* set SYN,  AC0 bit */
-                            Jedec.GALAC0 = 1;
-
-                            break;
-                        }
```

Comment:
Deletion of this block of code looks like it might have unintended consequences for gal_type other than GAL16V8 or GAL20V8, i.e. GAL20RA10 or GAL22V10 - Jedec.GALSYN and Jedec.GALAC0 could be left uninitialized?

Reply:
After a second look, there is a block of code just below which will set 'modus' and initialize Jedec.GALSYN and Jedec.GALAC0, however 'modus' will be set to MODE1 instead of MODE2 and Jedec.GALAC0 will be set to 0 instead of 1.

Jedec.GALAC0 is used only for GAL16V8 and GAL20V8 (in jedec.c) so any difference in value for other GAL types should not cause a problem.

However the potential difference in 'modus' for GAL20RA10 and GAL22V10 looks like it would affect the subsequent loop which is setting PinType and TriCon based on 'modus'. I'm trying to understand if this possible change in behavior for GAL20RA10 and GAL22V10 is desirable.